src/loaders/tvg/tvgTvgLoadParser.cpp

```diff
@@ -114,9 +114,7 @@ static LoaderResult _parseCmpTarget(const char *ptr, const char *end, Paint *pai
     auto cmpBlock = _readBlock(ptr);
     if (cmpBlock.end > end) return LoaderResult::SizeCorruption;
 
-    if (paint->composite(unique_ptr<Paint>(_parsePaint(cmpBlock)), cmpMethod) != Result::Success) {
-        return LoaderResult::MemoryCorruption;
-    }
+    paint->composite(unique_ptr<Paint>(), cmpMethod);
```

Comment:
Undo the change. There it was ok.

Reply:
@mmaciola it's my intention. not so useful.

Reply:
please fix it: target for composite is missing there.
The previous code also handled nullptr from _parsePaint that here signifies an error.